api/app/hfi/hfi_calc.py

```diff
@@ -110,7 +107,7 @@ def generate_station_daily(raw_daily,  # pylint: disable=too-many-locals
         error=raw_daily.get('observationValidInd', None),
         error_message=raw_daily.get('observationValidComment', None),
         last_updated=datetime.fromtimestamp(raw_daily.get(
-            'lastEntityUpdateTimestamp') / 1000, tz=timezone.utc)
```

Comment:
Not sure if lastEntityUpdateTimestamp is mandatory or not? If it is - can switch to use [] and not set default of 0.

Reply:
According to WFWX, it is a required field we can rely on from their API (see thread): https://chat.developer.gov.bc.ca/channel/wildfire-wfwx?msg=BvE29dpw6JuXTmd3i

Might make it easier for us to also just make it required.